ECOSYSTEM.md

```diff
@@ -15,3 +15,4 @@ Plugins and tools maintained by the knex team are listed under [Core](#core) whi
 - [`knex-serverless-mysql`](https://github.com/MatissJanis/knex-serverless-mysql) Knex dialect for usage with [`serverless-mysql`](https://github.com/jeremydaly/serverless-mysql): persist database connection over multiple serverless (such as AWS Lambda) function invocations.
 - [`pg-mem`](https://github.com/oguimbal/pg-mem) An in-memory emulation of PostgreSQL instance, which can be used for writing blazingly fast tests. Has [adapter for Knex](https://github.com/oguimbal/pg-mem/wiki/Libraries-adapters#-knex).
 - [`sqlcommenter-knex`](https://github.com/google/sqlcommenter/tree/master/nodejs/sqlcommenter-nodejs/packages/sqlcommenter-knex) A [sqlcommenter](https://google.github.io/sqlcommenter/) plugin with [support for Knex](https://google.github.io/sqlcommenter/node/knex/) to augment SQL statements with comments that can be used later to correlate application code with SQL statements.
+- [`knex-firebird-dialect`](https://github.com/Tomas2D/knex-firebird-dialect) Knex dialect for a Firebird database.
```

Comment:
can you create a separate section on top for dialects?

Reply:
Updated